src/main/java/net/erv123/shadertoymc/commands/ShadertoyCommand.java

```diff
@@ -0,0 +1,81 @@
+package net.erv123.shadertoymc.commands;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.StringArgumentType;
+import me.senseiwells.arucas.utils.Util;
+import net.erv123.shadertoymc.util.ScriptUtils;
+import net.erv123.shadertoymc.util.ShaderUtils;
+import net.erv123.shadertoymc.ShadertoyMC;
+import net.minecraft.command.CommandSource;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.text.ClickEvent;
+import net.minecraft.text.Text;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Locale;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+import static net.minecraft.server.command.CommandManager.argument;
+import static net.minecraft.server.command.CommandManager.literal;
+
+public class ShadertoyCommand {
+	/*
+			/shadertoy
+				area
+					pos size
+					clear
+					display
+					pos1
+					pos2
+					size
+
+				shader
+					calculate
+					paste
+	 */
+	private ShadertoyCommand() {
+
+	}
+
+	public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+		dispatcher.register(literal("shadertoy")
+			.requires(source -> source.hasPermissionLevel(2))
+			.then(literal("run")
```

Comment:
why no stop command?

Reply:
You cannot 'stop' it with the new sandboxed implementation. And why would you ever want to stop? WorldEdit doesn't provide this and I don't see why you would.